docs/source/tutorials/multiple_pilots.ipynb

```diff
@@ -1,20 +1,420 @@
 {
```

Comment:
<p>Should we list all the available TaskManager 'scheduling' algorithms we support and then give an example of them below, extending the notebook? In production and for our publications we never used round robin, as it would be inefficient. I am assuming users are not likely to use it either. The one we used to most was back filling. Maybe we could also explain the pros/cons of each algorithm (or reference them) and add examples that could be useful to the users?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
added and updated, also rechecked the default scheduling algorithm - it is round robin.